### PR TITLE
GROOVIE: Add keymapper support

### PR DESCRIPTION
**Pull request by NabeelShabbir**

### CODE REVIEW

**Comment by bluegr** [comment]
@NabeelShabbir there's an unrelated commit for AGOS in this PR now

File: engines/groovie/metaengine.cpp [hunk 1]
```diff
@@ -188,6 +194,36 @@ int GroovieMetaEngine::getAutosaveSlot() const {
 	return GroovieEngine::AUTOSAVE_SLOT;
 }
 
+Common::KeymapArray GroovieMetaEngine::initKeymaps(const char *target) const {
+	using namespace Common;
+	using namespace Groovie;
+
+	Keymap *engineKeyMap = new Keymap(Keymap::kKeymapTypeGame, "groovie-engine", "Groovie engine");
```
Comment (OMGPizzaGuy) [inline engines/groovie/metaengine.cpp:201]: We might want localization with _("Groovie engine")

**Review by OMGPizzaGuy — APPROVED** [review]
Looks good to me!

**Comment by bluegr** [comment]
Nice work, thanks!

**Pull request merged**